src/aws-cpp-sdk-core/include/aws/core/client/AWSErrorMarshaller.h

```diff
@@ -53,6 +55,15 @@ namespace Aws
              */
             virtual AWSError<CoreErrors> FindErrorByName(const char* exceptionName) const;
             virtual AWSError<CoreErrors> FindErrorByHttpResponseCode(Aws::Http::HttpResponseCode code) const;
+
+            /**
+             * Unmarshall http error payload to an error object.
```

Comment:
looks like cspell doesnt like `Unmarshall`, looks like `Unmarshal` is correct spelling